common/desktop-exports

```diff
@@ -22,15 +22,34 @@ fi
 
 export SNAP_LAUNCHER_ARCH_TRIPLET=$ARCH
 
+if [ -d $SNAP/gnome-runtime ]; then
+  if [ ! -d $SNAP/gnome-runtime/usr ]; then
+    echo "You need to connect the snap to the gnome-runtime interface."
+    echo ""
+    echo "You can do this with those commands:"
+    echo "snap install gnome318-udt"
```

Comment:
Hum, I wonder if we can know from the $SNAP/meta/snap.yaml which default provider we expect and is not connected to grasp from there the correct snap to install instead of single casing this, making sense?